inference-engine/tests/functional/plugin/myriad/ngraph/operations/static_shape_broadcast.cpp

```diff
@@ -137,6 +147,22 @@ INSTANTIATE_TEST_CASE_P(smoke_NGraph, StaticShapeBroadcastExplicitTests, testing
         testing::ValuesIn(testExplicitStaticShapes))
 );
 
+TEST_P(StaticShapeBroadcastBidirectionalTests, CanValidateAndInferTypes) {
+    const auto shapeOf = std::make_shared<ngraph::opset3::ShapeOf>(m_tensorWithTargetShape);
+    std::shared_ptr<ngraph::vpu::op::StaticShapeBroadcast> op;
+    ASSERT_NO_THROW(op = std::make_shared<ngraph::vpu::op::StaticShapeBroadcast>(
+            m_tensor, shapeOf, ngraph::op::BroadcastType::BIDIRECTIONAL));
+    ASSERT_NO_THROW(std::make_shared<ngraph::Function>(
+            ngraph::OutputVector{op->output(0)},
+            ngraph::ParameterVector{m_tensor, m_tensorWithTargetShape}));
```

Comment:
I actually would like to see something like `ASSERT_EQ(m_tensorWithTargetShape->get_shape(), op->output(0).get_shape());`, but as targetShape is not equal to output in `BIDIRECTIONAL` case, you may extend `BroadcastNumpyShapes` to `BroadcastBidirectionalShapes` with the third shape which is actual output shape.

Reply:
I've added it